webapp/src/components/AppNav.js

```diff
@@ -5,32 +5,27 @@ import React from "react";
 import { Link, useLocation } from "react-router-dom";
 
 export default function AppNav() {
-  const { setAppNav, topNav } = useGlobalViewState();
-  const top = topNav?.clientHeight || 0;
+  const { setAppNav } = useGlobalViewState();
   return (
-    <div ref={setAppNav} className="app-nav d-flex flex-row sticky-top" style={{ top }}>
-      <AppLink to="/dashboard" label={t("titles:home")} style={{ borderRightWidth: 0 }} />
-      <AppLink
-        to="/mobility"
-        label={t("titles:mobility")}
-        style={{ borderRightWidth: 0 }}
-      />
-      <AppLink to="/food" label={t("food:title")} style={{ borderRightWidth: 0 }} />
```

Comment:
Nice fix on the styles.

Reply:
Little things like this are important since 1) they build up and add complexity over time and 2) fixing them increases your expertise with the code. It's excellent you are fixing them when you see them.

Reply:
Agreed. I hunt *styles* for a living 😄